ppr-ui/prettier.config.js

```diff
@@ -1,14 +1,16 @@
 module.exports = {
   endOfLine: 'lf',
   jsxBracketSameLine: false,
-  printWidth: 80,
+  printWidth: 120,
```

Comment:
This is probably a good change, is this why it broke 3 lines of code into 5?
Does this fix that?

Reply:
Yea, this helps with that. Now, if you use prettier with this config, it won't (shouldn't) break the lines unless over 120 chars.